masu/database/provider_billing_source_db_accessor.py

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Accessor for Provider Billing Source from koku database."""
+
+
+from masu.database.koku_database_access import KokuDBAccess
+
+
+class ProviderBillingSourceDBAccessor(KokuDBAccess):
+    """Class to interact with the koku database for Provider Billing Source Data."""
+
+    def __init__(self, billing_source_id, schema='public'):
+        """Establish Provider Billing Source database connection."""
+        super().__init__(schema)
+        self._billing_source_id = billing_source_id
+        self._billing_source = self.get_base().classes.api_providerbillingsource
+
+    def _get_db_obj_query(self):
+        """Get the query for the given item."""
+        obj = self.get_session().query(self._billing_source).filter_by(id=self._billing_source_id)
+        return obj
+
+    def get_uuid(self):
+        """Return the provider uuid."""
+        obj = self._get_db_obj_query().first()
+        return obj.uuid
+
+    def get_bucket(self):
+        """Return the provider resource name."""
```

Comment:
Is it returning a bucket name (e.g. "my-bucket-name") or an ARN (e.g. "aws::s3::whatever/my-bucket-name")?

Docstring should include an example of the return value.

Reply:
Yes, I'll update the docstrings.  Good call.

Reply:
Perhaps we can agree on a docstring style so we all know what to expect. #codingstandards